locales/en/translation.json

```diff
@@ -216,32 +216,32 @@
   "hours_timeago": "hours",
   "How do you describe their diabetes?": "",
   "How do you describe your diabetes?": "",
-  "html.basics-no-uploaded-data": "<0>The Basics view shows a summary of your recent device activity, but it looks like you haven't uploaded device data yet.</0><1>To see the Basics, <1>upload</1> some device data.</1><2>If you just uploaded, try <1>refreshing</1>.</2>",
+  "html.basics-no-uploaded-data": "<0>The Basics view shows a summary of your recent device activity, but it looks like your device hasn't sent data yet.</0><1><1>Check</1> your device is activated.</1><2>If you just checked it, try <1>refreshing</1>.</2>",
   "html.browser-warning-download-text": "Or download Chrome <1>here</1>.",
   "html.browser-warning-text": "<0>Copy and paste</0><1></1><2>into Chrome.</2>",
-  "html.donate-form-explainer": " You own your data. Read all the details about Tidepool's Big Data Donation project <1>here</1>.",
+  "html.donate-form-explainer": " You own your data. Read all the details about YourLoops's Big Data project <1>here</1>.",
   "html.emailverification-instructions": "<0>Keeping your data private and secure is important to us!</0><1><0>Please click the link in the email we just sent you at<1></1><2><0>{{sent}}</0></2><3></3>to verify and activate your account.</0></1>",
-  "html.patient-delete-account": "If you are sure you want to delete your account, <1>send an email</1> to support@tidepool.org and we take care of it for you.",
+  "html.patient-delete-account": "If you are sure you want to delete your account, <1>send an email</1> to support@diabeloop.fr and we take care of it for you.",
   "html.patient-info-fullname": "<0>{{fullName}}</0> (edit in<2>account</2>)",
-  "html.patientdata-uploaded-message": "<0>To see your data, you’ll need the Tidepool Uploader</0><1></1><2>Already have the Tidepool Uploader? Launch it <1>here</1></2><3>To upload Dexcom with iPhone, get <1>Tidepool Mobile</1></3><4>Already uploaded? <1>Click to reload.</1><2></2><3>Need help?</3> Email us at <5>support@tidepool.org</5> or visit our <7>help page</7>.</4>",
+  "html.patientdata-uploaded-message": "<0>To see your data, you’ll need a Diabeloop device</0><1></1><2>Already have a Diabeloop device? </2><4><3>Need help?</3> Email us at <5>support@diabeloop.fr</5> or visit our <7>help page</7>.</4>",
```

Comment:
Pareil j'aimerais bien savoir où se situe ce message et quand il apparait car ça pourrait être aussi " to see your data, you'll need an internet connection/ a chrome web browser"